### ISSUE TITLE
Add armhf.deb for the pinebook pro

### ISSUE THREAD

**Issue by pinebookfan** [MrAlex94/Waterfox#1438]
Hello,

Can someone create a armhf.deb for the pinebook pro?
I don't want to use the default firefox browser or chromium.
Should be possible :)

Thanx!

**Comment by hawkeye116477** [comment]
What distro have you?

**Comment by pinebookfan** [comment]
Debian 9 Stretch armhf 32bit with libc6 2.24
Thats the default os for the pinebook pro

**Comment by pinebookfan** [comment]
AppImageLauncherNow works on Pinebook Pro:
https://github.com/TheAssassin/AppImageLauncher/releases

So an armhf appimage is now possible! Make it's easier to make :)

**Comment by pinebookfan** [comment]
Maybe!

**Comment by hawkeye116477** [comment]
Are you want Classic or Current?

**Comment by pinebookfan** [comment]
Current Thanx you so much!

**Comment by hawkeye116477** [comment]
@pinebookfan I have good news, finally succesfully build Current for amv7l/armhf with few patches.

To install it, just do:

> 
> echo 'deb http://download.opensuse.org/repositories/home:/hawkeye116477:/waterfox/Debian_9.0/ /' > /etc/apt/sources.list.d/home:hawkeye116477:waterfox.list
> wget -nv https://download.opensuse.org/repositories/home:hawkeye116477:waterfox/Debian_9.0/Release.key -O Release.key
> apt-key add - < Release.key
> apt-get update
> apt-get install waterfox-current-kpe

**Comment by pinebookfan** [comment]
Install works, but app isn't starting on pinebook pro...
Thanx!!!

**Comment by hawkeye116477** [comment]
@pinebookfan What shows when you type `waterfox-current` from terminal?

**Comment by pinebookfan** [comment]
bus error

**Comment by stevenpusser** [comment]
Isn't the Pinebook Pro running a 64-bit ARM CPU?
https://wiki.pine64.org/index.php/Pinebook_Pro#CPU_Architecture
Debian calls that arm64, not armhf, while the OBS calls it "aarch64".

**Comment by hawkeye116477** [comment]
https://github.com/mrfixit2001/debian_desktop

`Installing Iceweasel (Firefox) via apt will not work - you need a custom deb (armhf) or compile from source`

**Comment by hawkeye116477** [comment]
So seems that author of default image `https://github.com/mrfixit2001/debian_desktop` misleading.

Update: I set-up emulator for testing. I wanted to build Current now on OBS, but required packages are hanged on `blocked` state, so I'll probably need to try another day.

**Comment by pinebookfan** [comment]
The Pinebook Pro is running a 64-bit ARM CPU, that's right.
True if you use another os it's arm64 or aarch64. 
The original Pinebook Pro image use  armhf because of wide vine for Videostream-Services.
So the default OS runs armhf. trust me, only the manjaro runs smooth but the default armhf is the best to use. Only option if you want to use Netflix :)

**Comment by hawkeye116477** [comment]
@pinebookfan Remove package and try again with:
```
apt-get update
apt-get install waterfox-current-kpe
```

**Comment by pinebookfan** [comment]
not working, bus error again

**Comment by pinebookfan** [comment]
not working, bus error again

**Comment by hawkeye116477** [comment]
Ok, so I don't know what is cause. I used same patches for armhf as Debian and gcc compiler (tried before with Clang, but got segfault). Tried run it even on emulator, had also different error than you, but same was even with Firefox ESR and Chromium from repository. However when I tested before for aarch64, Firefox worked great. So maybe with aarch64, it would be easier.

Thus, I give up at this point, if you want for armhf, then you'll probably need to find more experienced person.
